Misc/NEWS.d/next/Library/2024-07-13-06-23-24.gh-issue-121245.RfOgf4.rst

```diff
@@ -0,0 +1,3 @@
+Simplify handling of the history file in ``site.register_readline()``
```

Comment:
Note that I'm pretty sure this suggestion will break buildbots while `site.register_readline` is undocumented. Document that before adding this.
```suggestion
Simplify handling of the history file in :func:`site.register_readline`
```

Reply:
I think this out of the scope for this pr.